src/test/resources/names.pan

```diff
@@ -0,0 +1,12 @@
+object template names;
+
+include 'quattor/types/metadata';
+# wait for https://github.com/quattor/template-library-core/pull/127
+#bind '/metadata' = structure_metadata;
+
+prefix "/metadata/template/branch";
+"name" = "mybranch";
+"type" = "production";
```

Comment:
Aquilon uses sandbox or domain here so it would be better to use one of those two unless other community members are using this field in a different way?